packages/editor-ui/src/components/forms/ResourceFiltersDropdown.vue

```diff
@@ -136,9 +142,31 @@ onBeforeMount(async () => {
 </template>
 
 <style lang="scss" module>
+@use 'n8n-design-system/css/mixins/breakpoints' as mixins;
+
 .filter-button {
 	height: 40px;
 	align-items: center;
+
+	.filter-button-count {
+		margin-right: var(--spacing-4xs);
+
+		@include mixins.breakpoint('xs-only') {
+			margin-right: 0;
+		}
+	}
+
+	.filter-button-text {
+		@media screen and (max-width: 480px) {
+			text-indent: -10000px;
+		}
+	}
+
+	@include mixins.breakpoint('xs-only') {
+		:global(span + span) {
```

Comment:
Is this to disable button label margin if there's both icon or label? 🤔 Could you please drop a small comment to clarify

Reply:
Added a comment